src/actions/actions.ts

```diff
@@ -43,6 +43,42 @@ const compareKeypressSequence = function (one: string[], two: string[]): boolean
   return true;
 };
 
+const adjustVimSelectionPosition = function(vimState: VimState, start: Position, end: Position): any {
```

Comment:
Thanks for the naming suggestion, it's always pain for non-English speaker like me :) The strict return type suggestion is also good, will change it later.